led_strip/examples/led_strip_spi_ws2812/main/led_strip_spi_ws2812_main.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Unlicense OR CC0-1.0
+ */
+#include <stdio.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "led_strip.h"
+#include "esp_log.h"
+#include "esp_err.h"
+
+// GPIO assignment
+#define LED_STRIP_BLINK_GPIO  14
+
+// LED numbers in the strip
+#define LED_STRIP_LED_NUMBERS 24
+
+//#define CONFIG_BLINK_LED_RMT
+#define CONFIG_BLINK_LED_SPI
+
+static const char *TAG = "example";
+
+#define LED_STRIP_SPI_RES_HZ  (2.5 * 1000 * 1000)
+// 2.5MHz resolution, 1 tick = 0.4us (led strip needs a high resolution)
+
+// LED Strip object handle
+led_strip_handle_t led_strip;
+
+void configure_led()
+{
+    // LED strip general initialization, according to your led board design
+    led_strip_config_t strip_config = {
+        .strip_gpio_num = LED_STRIP_BLINK_GPIO,   // The GPIO that connected to the LED strip's data line
+        .max_leds = LED_STRIP_LED_NUMBERS,        // The number of LEDs in the strip,
+        .led_pixel_format = LED_PIXEL_FORMAT_GRB, // Pixel format of your LED strip
+        .led_model = LED_MODEL_WS2812,            // LED strip model
+        .flags.invert_out = false,                // whether to invert the output signal
+    };
+
+    // LED strip backend configuration: SPI
+    led_strip_spi_config_t spi_config = {
+        .clk_src = SPI_CLK_SRC_DEFAULT,        // different clock source can lead to different power consumption
+        .resolution_hz = LED_STRIP_SPI_RES_HZ, // SPI counter clock frequency
+        .flags.with_dma = true,               // DMA feature is available on ESP target like ESP32-S3
```

Comment:
the comments seems like copied from the RMT example... In our chip, SPI peripheral always support DMA and non-DMA. But we recommend to use the DMA in the led_strip application. Please update the comments accordingly.